dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -34,11 +36,6 @@ wil::unique_event& GetWaitHandleForArgs()
     return g_waitHandleForArgs;
```

Comment:
Why do we have this static? Can we move it to be a member variable?

Reply:
@pmpurifoy

Reply:
waitHandle is used in GetNotificationEventArgs.h to receive the args from background. I have an idea which we move PushDeserialize in PushNotificationManager which then we could turn waitHandle that into a member variable

Reply:
We can make this a bug for improvement.